app/components/view_component/form/rich_text_area_component.rb

```diff
@@ -1,10 +1,14 @@
 # frozen_string_literal: true
 
-if defined?(ActionView::Helpers::Tags::ActionText)
-  module ViewComponent
-    module Form
-      class RichTextAreaComponent < FieldComponent
+module ViewComponent
+  module Form
+    class RichTextAreaComponent < FieldComponent
+      if defined?(ActionView::Helpers::Tags::ActionText)
         self.tag_klass = ActionView::Helpers::Tags::ActionText
+      else
+        def call
+          raise NotImplementedError, "#{self.class} requires action_text to be loaded"
+        end
```

Comment:
@Spone I'm finally not sure about having this since the helper in builder will not be defined if `action_text` not loaded.

Maybe we should just keep the if part, and remove the else one.

Reply:
Sounds good!

Reply:
Sounds good!

Reply:
Ok, I'll do the change then